zproject/backends.py

```diff
@@ -1018,6 +1027,12 @@ class SAMLAuthBackend(SocialAuthMixin, SAMLAuth):
     standard_relay_params = ["subdomain", "multiuse_object_key", "mobile_flow_otp",
                              "next", "is_signup"]
     REDIS_EXPIRATION_SECONDS = 60 * 15
+    name = "saml"
+    # Organization which go through the trouble of setting up SAML are most likely
+    # to have it as their main authentication method, so it seems appropriate to have
+    # SAML buttons at the top.
+    sort_order = 9999
+    display_logo = "/static/images/landing-page/logos/saml-icon.png"
```

Comment:
Would it be reasonable to support a no-logo button format, and have that be the default for SAML?  I expect it wouldn't be hard to to do CSS for that, and there may be organizations that prefer that to either using a non-standard logo or needing to include their own.

Reply:
Made a simple change for that in a new commit, I'll post more thoughts in a separate comment below.